tests/conftest.py

```diff
@@ -48,7 +56,9 @@ def fx_raw_authors() -> list[dict[str, Any]]:
 
 
 @pytest.fixture(name="authors")
-def fx_authors(raw_authors: list[dict[str, Any]]) -> list[authors.Author]:
+def fx_authors(
```

Comment:
maybe even these, and any tests that depend on them should be moved into the `sqlalchemy` test directory..